cmd/gotohelm/main.go

```diff
@@ -24,13 +23,21 @@ func main() {
 		panic(err)
 	}
 
+	pkgs = append(pkgs, gotohelm.ShimsPkg)
```

Comment:
This logic should like within the `Transpile` function. Why was it moved out? Seems like it would have worked without issue as it was 🤔

Reply:
It can but the file will be generated per package. In Redpanda chart case you will find shim in 3 folders. I believe the definition of the function/template can not duplicate within one helm chart. That's why I treat bootstrap packae as a separated entitiy